feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/stats_nl80211.c

```diff
@@ -597,18 +598,26 @@ int nl80211_scan_dump(struct nl_call_param *nl_call_param)
 	if (!msg)
 		return -1;
 
-	return unl_genl_request(&unl, msg, nl80211_scan_dump_recv, nl_call_param);
+	return unl_genl_request(&unl_req, msg, nl80211_scan_dump_recv, nl_call_param);
 }
 
 int stats_nl80211_init(void)
 {
-	if (unl_genl_init(&unl, "nl80211") < 0) {
+	if (unl_genl_init(&unl_req, "nl80211") < 0) {
```

Comment:
Should unl_req also have the increased buffer size?

Reply:
I believe it is not needed. unl_req is used for netlink request and receive messages as opposed to unl_notify which is used to receive notifications. It is the notifications buffer that could grow if there is a delay in reading or if we have a burst of events as seen with connecting multiple clients at one go.